### PR TITLE
docs(pie-docs): DSW-1624 update pie-webc documentation for pie-docs

### PR DESCRIPTION
**Pull request by leksaBoiko**
## Describe your changes (can list changeset entries if preferable)

**[Updated]** - pie-webc documentation for consumers

## Updated WIKI pages: 
[Vue Integration Guide](https://github.com/justeattakeaway/pie/wiki/PIE-Web-Components-%E2%80%90-Vue-Integration-Guide)
[Nuxt 3](https://github.com/justeattakeaway/pie/wiki/PIE-Web-Components-%E2%80%90-Nuxt-3)

## Author Checklist (complete before requesting a review)
- [x] I have performed a self-review of my code
- [x] I have added thorough tests where applicable (unit / component / visual)
- [x] I have reviewed the `PIE Docs` PR preview
- [x] I have reviewed visual test updates properly before approving
- [x] If changes will affect consumers of the package, I have created a changeset entry.

### Reviewer 1 - @xander-marjoram 
- [x] I have reviewed the `PIE Docs` PR preview
- [x]  I have reviewed the `PIE Wiki` pages
- [x] If there are visual test updates, I have reviewed them

### Reviewer 2 @kevinrodrigues 
- [x] I have reviewed the `PIE Docs` PR preview
- [x] I have reviewed the `PIE Wiki` pages
- [x] If there are visual test updates, I have reviewed them

### CODE REVIEW

**Review by xander-marjoram — COMMENTED** [review]
Great stuff! Just one comment.

Please can you update the code page template (`pie-docs/src/components/template/code/code.md`) with the two version badges?

**Review by xander-marjoram — APPROVED** [review]

**Review by kevinrodrigues — DISMISSED** [review]

File: apps/pie-docs/src/components/icon-button/code/code.md [hunk 1]
```diff
@@ -124,10 +129,19 @@ import { PieIconButton } from '@justeattakeaway/pie-icon-button';
 
 For React Applications:
 
+```jsx
+import { PieIconButton } from '@justeattakeaway/pie-webc/react/icon-button.js';
+import { IconClose } from '@justeattakeaway/pie-icons-webc';
```
Comment (xander-marjoram) [inline apps/pie-docs/src/components/icon-button/code/code.md:134]: Actually I just noticed, I think this should be `/dist/react/IconClose.js` and then the same but without the file extension for React 13

File: apps/pie-docs/src/components/text-input/code/code.md [hunk 2]
```diff
@@ -116,6 +120,25 @@ import { PieTextInput } from '@justeattakeaway/pie-text-input/dist/react';
 </PieTextInput>
 ```
 
+```jsx
+// React templates (using Next 13)
+import { PieTextInput } from '@justeattakeaway/pie-text-input/dist/react';
```
Comment (xander-marjoram) [inline apps/pie-docs/src/components/text-input/code/code.md:125]:
```suggestion
import { PieTextInput } from '@justeattakeaway/pie-webc/react/text-input';
```

File: apps/pie-docs/src/components/tag/code/code.md [hunk 3]
```diff
@@ -105,19 +109,24 @@ For Native JS Applications, Vue, Angular, Svelte etc.:
...
-```js
+```jsx
+import { PieTag } from '@justeattakeaway/pie-webc/react/tag.js';
+
+<PieTag>Label</PieTag>
+```
+
+```jsx
 // React templates (using Next 13)
 import { PieTag } from '@justeattakeaway/pie-tag/dist/react';
```
Comment (xander-marjoram) [inline apps/pie-docs/src/components/tag/code/code.md:127]: I think this needs updating to `pie-webc/react/tag`

Comment (xander-marjoram) [reply apps/pie-docs/src/components/tag/code/code.md:127]: And a couple of other components, I'll let you go through them all 👍🏻

File: apps/pie-docs/src/components/text-input/code/code.md (hunk 2 above)
Comment (leksaBoiko) [reply apps/pie-docs/src/components/text-input/code/code.md:125]: Next 13 apps requiring SSR should import the components individually, shouldn't they?

**Comment by xander-marjoram** [comment]
The code, deployments and visual regression tests all look good to me!
Just a few comments about the wiki articles:

1. The Vue integration guide includes a couple of examples for React apps. These are probably unneceessary and should live in another page.
2. In the Nuxt 3 integration guide, could you add `nuxt-ssr-lit` to the installation example in Step 1?
3. In the Nuxt 3 integration guide, one of the code snippets uses the React import path. I also don't think it's necessary to destructure the imports (`import { PieButton }`) because those imported objects aren't referenced.

**Comment by leksaBoiko** [comment]
> The code, deployments and visual regression tests all look good to me! Just a few comments about the wiki articles:
> 
> 1. The Vue integration guide includes a couple of examples for React apps. These are probably unneceessary and should live in another page.
> 2. In the Nuxt 3 integration guide, could you add `nuxt-ssr-lit` to the installation example in Step 1?
> 3. In the Nuxt 3 integration guide, one of the code snippets uses the React import path. I also don't think it's necessary to destructure the imports (`import { PieButton }`) because those imported objects aren't referenced.

yep, added changes to Wiki pages

**Review by xander-marjoram — APPROVED** [review]

**Review by kevinrodrigues — APPROVED** [review]

**Review by xander-marjoram — APPROVED** [review]

**Review by kevinrodrigues — APPROVED** [review]

**Pull request merged**